Feature/upgrade firmachain v0.5.0

LittleBeaverDev opened the pull request:
- Update firma-js version to v0.3.1
- Apply obfuscation
- Update readme

LittleBeaverDev committed:
update: firma-js version

LittleBeaverDev committed:
update: update build config

LittleBeaverDev committed:
update: set the amount of tokens sent per request to 1 FCT

LittleBeaverDev committed:
feat: show result data on success

- After chain upgrade, the result data is not sent via 'rawLog'
- Removed install states

LittleBeaverDev committed:
fix: block invalid address

LittleBeaverDev committed:
feat: block token request when user hold more than 10 fct

LittleBeaverDev committed:
chore: update firma-js v0.3.1

- Update firma-js version to v0.3.1
- Update readme

LittleBeaverDev committed:
misc: change 'cannot claim' message

- Change change 'cannot claim' message
- Add port config for preview

lincheoll commented:
![image](https://github.com/user-attachments/assets/ea29472d-a6b9-45cb-ba7c-b7222dcf75fe)
please remove this part.

LittleBeaverDev committed:
update: remove gas/result json viewer

- Remove gas/result json viewer
- Add preview method
- Fix config lint issue
- Set copyright year to 2025

lincheoll reviewed (approved):
LGTM
